assets/dev/js/editor/utils/helpers.js

```diff
@@ -418,63 +418,39 @@ module.exports = {
 	},
 
 	isActiveControl: function( controlModel, values ) {
-		let condition,
-			conditions;
-
-		// TODO: Better way to get this?
-		if ( _.isFunction( controlModel.get ) ) {
-			condition = controlModel.get( 'condition' );
-			conditions = controlModel.get( 'conditions' );
-		} else {
-			condition = controlModel.condition;
-			conditions = controlModel.conditions;
-		}
+		const condition = controlModel.condition || controlModel.get?.( 'condition' );
+		let conditions = controlModel.conditions || controlModel.get?.( 'conditions' );
+
+		// If there is a simple condition, convert it to a complex condition.
+		if ( condition ) {
+			const terms = [];
+
+			Object.entries( condition ).forEach( ( [ conditionName, conditionValue ] ) => {
+				const conditionNameParts = conditionName.match( /(\w+(?:\[\w+])?)?(!?)$/i ),
```

Comment:
Add an example for a `conditionName` that matches the regex

Reply:
> Add an example for a `conditionName` that matches the regex

As a comment in the code? Alright.